modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/Main.scala

```diff
@@ -65,22 +65,25 @@ object Main extends IOApp {
     import env._
 
     Stream.eval_(Manifest.initialize[IO](cli.config.storage, cli.dryRun, env.blocker)) ++
-      DataDiscovery.discover[IO](cli.config, env.state)
+      DataDiscovery
+        .discover[IO](cli.config, env.state)
```

Comment:
Now that `discover` can raise an error, don't we want to use `handleErrorWith` on it ?

Reply:
So this is the interesting one. I didn't do this because I assumed that the very last catch will do it: https://github.com/snowplow/snowplow-rdb-loader/pull/460/files/a781c65c5f91342489b3879ae0e45e06243449be..de57ed9f1e470c568b886323afebbd20db5b3cf6#diff-316cf91a44bc5b638963fa8c24bec5a5bef36e0e037e9967ba9e9adaed631d62R43-R46

On the other hand, in 1.0.0 we've never seen Sentry working out in our QA, so I'd like to double-check that it actually works.